third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -3421,7 +3421,7 @@ int AXPlatformNodeWin::MSAARole() {
       return ROLE_SYSTEM_TITLEBAR;
 
     case ax::mojom::Role::kToggleButton:
-      return ROLE_SYSTEM_PUSHBUTTON;
+      return ROLE_SYSTEM_CHECKBUTTON;
```

Comment:
Is check button basically a check box? Why do we choose this over push button?

Reply:
It is the same role as for a checkbox. In [#flutter/fluter/111525](https://github.com/flutter/flutter/issues/111525) we are currently of the opinion that it is the most appropriate role, as it is for interactable widgets with toggleable state, and the [materials design guidelines](https://material.io/components/switches#usage) recommend their use in place of checkboxes on mobile

Reply:
thanks for explanation. SGTM